File: android/api-tests/src/test/java/com/revenuecat/api_tests/java/OfferingsMapperApiTests.java
package com.revenuecat.api_tests.java.mappers;

import com.revenuecat.purchases.Offerings;
import com.revenuecat.purchases.hybridcommon.mappers.OfferingsMapperKt;

import java.util.Map;

@SuppressWarnings("unused")
class OfferingsMapperApiTests {
    private void checkMap(Offerings offerings) {
        Map<String, Object> map = OfferingsMapperKt.map(offerings);
    }
}

File: android/src/test/java/apitests/kotlin/com/revenuecat/purchases/hybridcommon/mappers/OfferingsMapperApiTests.kt
package apitests.kotlin.com.revenuecat.purchases.hybridcommon.mappers

import com.revenuecat.purchases.Offerings
import com.revenuecat.purchases.hybridcommon.mappers.map

@Suppress("unused", "UNUSED_VARIABLE")
private class OfferingsMapperApiTests {
    fun checkMap(offerings: Offerings) {
        val map: Map<String, Any?> = offerings.map()
    }
}
